README.md

```diff
@@ -5,44 +5,28 @@ Simple and efficient **simulation software for high-resolution atomic force micr
 
 ## Installation
 
-To install the latest version of PPM, run:
+The standard way of installing `ppafm` is:
 
 ```bash
 $ pip install ppafm
 ```
 
 This should install the package and all its dependencies.
-Once the installation is completed, the following commands should be available:
 
-- `ppafm-generate-elff` - command-line interface to generate electrostatic force field.
-- `ppafm-generate-elff-point-charges` - command-line interface to generate electrostatic force field using point charges.
-- `ppafm-generate-ljff` - command-line interface to generate Lennard-Jones force field.
-- `ppafm-relaxed-scan` - command-line interface to run a scan of the sample with the probe particle.
-- `ppafm-plot-results` - command-line interface to plot the results of the simulation.
-- `ppafm-gui` - GUI application for interactive simulation of AFM images. Requires some additional dependencies, see below.
+The most up-to-date installation guide can be found on the [dedicated wiki page](https://github.com/Probe-Particle/ppafm/wiki/Install-ppafm).
 
-Alternatives (Docker, Anaconda) and some notes on different platforms (Linux, MacOS, Windows) can be found on the wiki: [Additional installation instructions](https://github.com/Probe-Particle/ppafm/wiki/Additional-installation-instructions)
+## CLI
```

Comment:
My comment here, is that people which are starting with coding do not know, what CLI means, I would use full name here.

Reply:
thanks, done in [c961071](https://github.com/Probe-Particle/ppafm/pull/183/commits/c961071244888a1cbf8871b4ea7c7fb106d3dbc6)